1-exercises/H-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ let everyone = [
   "Swathi",
 ];
 
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive =everyone.slice (0,5)
+let lastFive = everyone .slice (2,7)
```

Comment:
Be sure your spacing is consistent
```suggestion
let firstFive = everyone.slice(0,5)
let lastFive = everyone.slice(2,7)
```